src/docs/theory/failure_modes.md

```diff
@@ -3,16 +3,53 @@ title: "Failure Modes"
 featured: ../images/featured/theory.png
 ---
 
-## WORK IN PROGRESS NOTES
-
-- Typical failure modes of software engineering
-- AuxEng Doing the hard work, building dependency
-- Not investing upfront in scaffolding
-- Disconnect between team and leadership
-  - Leaders want AuxEng, but the team doesn't
-- AuxEng feeling like an outsider
-- Lack of improvements to Platform layer
-- Lack of psychological safety in retros
-  - Welcome and reward critical feedback
-- Being just "embedded engineering"
-- Bad ratio of AuxEng to team members (1to6 is upper bound)
+TODO:
+* how to identify, avoid these situations
+
+An auxiliary engagement is a hopeful endeavor; there is no guarantee of success.
+An engagement can suffer from a number of failure modes, some easier to identify
+than others. The failure modes are typical of all software engineering, but can
+be especially potent in this context given the short timeline of an engagement.
+
+1. **Auxiliary Engineer does all the hard work**  
+    If the embedded engineer finds themselves solving hard problems in isolation,
+    without opportunities to pair, teach, or share, this is a cause for concern.
+    The embedded engineer should be pairing on large problems and sharing
+    decision-making  with the team, not operating as a contractor.
+
+1. **Auxiliary Engineer does not feel welcome**  
+    If the embedded engineer does not feel psychologically safe on the team, this
+    is a cause for concern. The embedded engineer should feel able to share their
+    opinions openly during code reviews, retros, and other communication with
+    team members or leadership. If this is not the case, this seriously affects
+    the embedded engineers ability to affect change within the team by limiting
+    the engineers ability to provide negative feedback. Negative feedback can be
+    instrumental in affecting change of a bad-habit type behavior.
+
+1. **A disconnect between the team and leadership**  
+    If there is a misunderstanding or misalignment between the team and
+    leadership, this can undermine the stability of the engagement. This could
+    take the form of excited leadership with under-engaged team, or a team with
+    one goal in mind and leadership with another. This can result in a team that
+    commits to a large idea on paper but does not execute, and cancelled
+    partnerships mid-engagement.
+
+1. **Lack of improvements to Platform layer**  
```

Comment:
```suggestion
1. **Lack of improvements to platform layer**  
```

given that later and elsewhere you mention the Platfrom as a (p)latform, I think it should be lowercase here!